src/components/Property/Types/StringProperty.tsx

```diff
@@ -1,27 +1,28 @@
+import { Paper, Text } from '@mantine/core';
+
 import { StringInput } from '@/components/Input/StringInput';
-import { PropertyLabel } from '@/components/Property/PropertyLabel';
 
-interface Props {
-  name: string;
-  description: string;
-  disabled: boolean;
+import { ConcretePropertyBaseProps } from '../types';
+
+interface Props extends ConcretePropertyBaseProps {
   setPropertyValue: (newValue: string) => void;
   value: string;
 }
 
-export function StringProperty({
-  name,
-  description,
-  disabled,
-  setPropertyValue,
-  value
-}: Props) {
+export function StringProperty({ name, disabled, setPropertyValue, value }: Props) {
+  if (disabled) {
+    return (
+      <Paper px={'sm'} py={5}>
+        <Text size={'sm'}>{value}</Text>
+      </Paper>
+    );
+  }
   return (
     <StringInput
       disabled={disabled}
```

Comment:
Why don't we want to use the string input if disabled? 
If we are to keep it this way we can remove line 22